Sources/Sentry/SentryNetworkTracker.m

```diff
@@ -57,6 +58,11 @@ - (void)urlSessionTaskResume:(NSURLSessionTask *)sessionTask
         }
     }
 
+    NSNumber *intercepted = [NSURLProtocol propertyForKey:SENTRY_INTERCEPTED_REQUEST
+                                                inRequest:[sessionTask currentRequest]];
+    if (intercepted != nil && [intercepted boolValue])
+        return;
```

Comment:
`m`: Why do we return here if the request was intercepted? Can we please also add tests for this?

Reply:
Since we swizzle the URLSessionTask resume, we would end up with a double span for each request since we also use SessionDataTask as proxy instead of implementing the HTTP protocol.

Reply:
Again can we add this info as a comment 😁